src/App.css

```diff
@@ -6,6 +6,14 @@
   margin: 40px 0 0;
 }
 
+.dg_icon-link .dg_popular_star {
+  color: gray;
+}
+
 .dg_icon-link:hover .dg_icon-link_footer-text {
   color: white;
 }
+
+.dg_icon-link:hover .dg_popular_star {
```

Comment:
not going to block, but this can be simplified to 

```css
.dg_icon-link:hover .dg_icon-link_footer-text,
.dg_icon-link:hover .dg_popular_star {
  color: white;
}
```

Reply:
done